oauth2-tests/build.gradle

```diff
@@ -45,7 +45,7 @@ cargo {
         }
         startStopTimeout = 240000
         containerProperties {
-            property 'cargo.start.jvmargs', '-Dspring.profiles.active=oauth'
+            property 'cargo.start.jvmargs', '-Dfineract.security.basicauth.enabled=false -Dfineract.security.oauth.enabled=true -Dfineract.security.2fa.enabled=false'
```

Comment:
Ie the environment is hardcoded here and these are the values used - correct?

Reply:
Correct, exactly the same behavior as before, just with properties instead of profiles.